### PR TITLE
Formatted types/k* packages with dprint

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
> 👉 Partial implementation of https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/65993. If you're seeing this and are surprised it's happening, please read the discussion and give feedback! ❤️ 

Implements the dprint formatting changes on packages whose names start with `k`. We're splitting up the changes to make it a bit easier to apply & review them en masse - and, if we need later on, revert / bisect for issues. These changes generated with roughly:

```shell
git checkout dprint-onboarding -- .dprint.jsonc
time npx dprint fmt "types/k*/**/*.t*"
# Formatted 407 files.
# npx dprint fmt "types/k*/**/*.t*"  3.13s user 0.61s system 177% cpu 2.108 total
rm .dprint.jsonc
```

~CI failures are due to [DefinitelyTyped-tools > feat: remove formatting rules rules from dtslint](https://github.com/microsoft/DefinitelyTyped-tools/pull/699) not being ~merged~ included in CI yet.~ ✔️

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## karma-coverage-istanbul-reporter ([<kbd>unpkg</kbd>](https://unpkg.com/browse/karma-coverage-istanbul-reporter@latest/))
was missing the following properties:
1. reporter:coverage-istanbul
1. reporter:coverage-istanbul
## karma-image-snapshot ([<kbd>unpkg</kbd>](https://unpkg.com/browse/karma-image-snapshot@latest/))
was missing the following properties:
1. framework:snapshot-jasmine
1. launcher:SnapshotLauncher
1. launcher:SnapshotHeadlessLauncher
1. markTouchedFile
1. reporter:outdated-snapshot
## katex ([<kbd>unpkg</kbd>](https://unpkg.com/browse/katex@latest/))
was missing the following properties:
1. katex
## kbm-robot ([<kbd>unpkg</kbd>](https://unpkg.com/browse/kbm-robot@latest/))
was missing the following properties:
1. mouseWheel
## keccak ([<kbd>unpkg</kbd>](https://unpkg.com/browse/keccak@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'keccak'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
## keep-network__tbtc.js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/keep-network__tbtc.js@latest/))
was missing the following properties:
1. BitcoinHelpers
## kefir ([<kbd>unpkg</kbd>](https://unpkg.com/browse/kefir@latest/))
was missing the following properties:
1. dissableDeprecationWarnings
1. Kefir
## kerberos ([<kbd>unpkg</kbd>](https://unpkg.com/browse/kerberos@latest/))
was missing the following properties:
1. Kerberos
1. version
1. processes
## keychain ([<kbd>unpkg</kbd>](https://unpkg.com/browse/keychain@latest/))
was missing the following properties:
1. executablePath
## keygrip ([<kbd>unpkg</kbd>](https://unpkg.com/browse/keygrip@latest/))
was missing the following properties:
1. sign
1. verify
1. index
## keypress.js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/keypress.js@latest/))
was missing the following properties:
1. keypress
## keysym ([<kbd>unpkg</kbd>](https://unpkg.com/browse/keysym@latest/))
was missing the following properties:
1. keyEvent
## keythereum ([<kbd>unpkg</kbd>](https://unpkg.com/browse/keythereum@latest/))
was missing the following properties:
1. scrypt
1. deriveKeyUsingScrypt
1. recover
## kiwicom__orbit-design-tokens ([<kbd>unpkg</kbd>](https://unpkg.com/browse/kiwicom__orbit-design-tokens@latest/))
was missing the following properties:
1. defaultTokens
## koa-bunyan-logger ([<kbd>unpkg</kbd>](https://unpkg.com/browse/koa-bunyan-logger@latest/))
was missing the following properties:
1. bunyan
## koa-hbs ([<kbd>unpkg</kbd>](https://unpkg.com/browse/koa-hbs@latest/))
was missing the following properties:
1. handlebars
1. configure
1. viewPath
1. templateOptions
1. extname

<details>
<summary>as well as these 51 other properties...</summary>
<p>partialsPath, contentHelperName, blockHelperName, defaultLayout, layoutsPath, locals, disableCache, partialsRegistered, cache, blocks, createRenderer, getLayoutPath, getLayoutTemplate, layoutTemplate, cacheLayout, loadLayoutFile, registerPartial, registerPartials, getTemplatePath, pathCache, content, block, create, handlebars, configure, viewPath, templateOptions, extname, partialsPath, contentHelperName, blockHelperName, defaultLayout, layoutsPath, locals, disableCache, partialsRegistered, cache, blocks, createRenderer, getLayoutPath, getLayoutTemplate, layoutTemplate, cacheLayout, loadLayoutFile, registerPartial, registerPartials, getTemplatePath, pathCache, content, block, create</p>
</details>

## kramed ([<kbd>unpkg</kbd>](https://unpkg.com/browse/kramed@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'kramed'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. options
1. setOptions
1. defaults
1. Parser

<details>
<summary>as well as these 7 other properties...</summary>
<p>parser, Renderer, Lexer, lexer, InlineLexer, inlineLexer, parse</p>
</details>

## kue ([<kbd>unpkg</kbd>](https://unpkg.com/browse/kue@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'kue'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. version
1. redis
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f331e84096d42f6dbf9228347b5082ddd1800476
</p>

File: types/kana2ipa/kana2ipa-tests.ts
Comment (na2na-p) [inline types/kana2ipa/kana2ipa-tests.ts:1]: Here is the package I added, Looks good to me.

**Review by SevenOutman — APPROVED** [review]
LGTM for `karma-sinon-chai`

**Review by 43081j — APPROVED** [review]

**Comment by yutak23** [comment]
LGTM for `kebabcase-keys`

**Review by johnnyreilly — APPROVED** [review]

**Comment by JoshuaKGoldberg** [comment]
I'll wait a day or two so that we don't publish _too_ many at once. Thanks for the reviews!

**Review by peterblazejewicz — APPROVED** [review]

**Review by wooogi123 — APPROVED** [review]

**Review by na2na-p — APPROVED** [review]

**Comment by JoshuaKGoldberg** [comment]
Ready to merge

**Pull request merged**